src/TgSelect/index.js

```diff
@@ -252,26 +256,28 @@ class TgSelect extends React.Component {
             onClick={this.handleClear}
           />
         )}
-        <Button
-          onClick={e => {
-            if (this.state.isOpen) {
-              e.stopPropagation();
+        {_noResults === undefined && (
```

Comment:
`_noResults` is just an aux variable. Not sure why this is added here. Only `noResults` should be used, and that will never be undefined anyways...